static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -0,0 +1,49 @@
+define([
+    'bean',
+    'fastdom',
+    'common/utils/$'
+], function (
+    bean,
+    fastdom,
+    $
+) {
+
+    function objToString(obj) {
```

Comment:
Just use `JSON.stringify`. If you call it like this:

```javascript
JSON.stringify(yourObject, true, 2);
```

it will indent to two spaces